docs/integrations/zapier.mdx

```diff
@@ -83,7 +83,7 @@ zapier push
 
 ### Try it out
 
-Finally, try it out! From the account where you deployed the integration, you can create a Zap and test the integration. It won't be publically available on Zapier until it's released, but you can share the integration with your users.
+Finally, try it out! From the account where you deployed the integration, you can create a Zap and test the integration. It will not be publicly available on the Zapier website until released, but you can share the integration with your users.
```

Comment:
I think it might have sounded a bit better as `until it's released` but up to you.